client/src/legacy/LeftAndMain.Panel.js

```diff
@@ -155,8 +155,6 @@ $.entwine('ss', function($) {
         this.setPersistedCollapsedState(!doExpand);
       }
 
-      // TODO Fix redraw order (inner to outer), and re-enable silent flag
-      // to avoid multiple expensive redraws on a single load.
       // if(!silent) {
```

Comment:
Please remove the commented-out code as well.

Reply:
Updated